src/main/java/pignlproc/markup/AnnotatingMarkupParser.java

```diff
@@ -385,6 +387,13 @@ public void nodesToText(List<? extends Object> nodes, Appendable buffer,
                                     tagName));
                             countingBuffer.append("\n\n");
                         }
+                        if ("b".equals(tagName)) {
+                            if (tagBegin < 500) {
```

Comment:
Just a question, is this for you to get boldforms occur in the first 500 chars of the page ?